homeassistant/components/lutron/__init__.py

```diff
@@ -64,19 +89,25 @@ def setup(hass: HomeAssistant, base_config: ConfigType) -> bool:
         "scene": [],
         "binary_sensor": [],
     }
+    host = config_entry.data[CONF_HOST]
+    uid = config_entry.data[CONF_USERNAME]
+    pwd = config_entry.data[CONF_PASSWORD]
 
-    config = base_config[DOMAIN]
-    hass.data[LUTRON_CONTROLLER] = Lutron(
-        config[CONF_HOST], config[CONF_USERNAME], config[CONF_PASSWORD]
-    )
+    def _connect() -> bool:
```

Comment:
This doesn't have to return something, it can just return None

Reply:
Since this function is only used in the executor job, you could just call this directly with the executor. Remind me to send an example

Reply:
@joostlek This is me reminding you for an example about the executor you mentioned.
Do feel free to propose the change as I'm pretty new to this.

Reply:
this appears to be a personal preference and not a requirement.  marking as resolved unless something more specific is provided